deepface/models/facial_recognition/Facenet.py

```diff
@@ -1711,3 +1731,36 @@ def load_facenet512d_model(
     # -------------------------
 
     return model
+
+
+def load_facenet512d_onnx_model(
+    url="https://github.com/ShivamSinghal1/deepface/releases/download/v1/facenet512_fp32.onnx",
+) -> Model:
```

Comment:
Is this function really retuning keras.models.Model?

Reply:
nope i have changed it to Any